recipes/r-misha/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "4.0.4" %}
+package:
+  name: 'r-misha'
+  version: '{{ version }}'
+
+source:
+  url: 'https://bitbucket.org/tanaylab/misha-package/get/96b4e57cfe08.zip'
```

Comment:
From what I gather, the following link should also work: <https://bitbucket.org/tanaylab/misha-package/get/4.0.4.zip>

So to avoid missing to update the link in future package updates, I would probably turn the url entry into:
```
  url: 'https://bitbucket.org/tanaylab/misha-package/get/{{ version }}.zip'
```

Reply:
Hi, 

I used the commit tag to get the latest versions. the repo has no tags, but I got the version number from the description in the source code. Hence, the proposed url wouldn't work, sadly.

Cheers
M

Reply:
If you check, there is a tag for this and the link I posted works. A bit hard to find in the download section:
https://bitbucket.org/tanaylab/misha-package/downloads/?tab=tags

But the commit tag works, as well. So this is just to prevent misses in the future when updating.

Reply:
I just merged the previous fix. If there's ever a need to update to a new version, I can update the URL.

m

Reply:
Yeah, no worries. If it builds, it should be fine.

Good luck with the downstream packages, and should they run into awkward troubles, keep in mind that it might have to do with the `rpaths:` entry in the `meta.yaml`:
https://github.com/bioconda/bioconda-recipes/pull/13410#issuecomment-458904991

Reply:
Thanks,

I know of the Rlibs, but in the template I used as shown in the downstream PR the libs weren't added, so I left them out too.